pkg/controller/deployment/deployment_controller.go

```diff
@@ -634,3 +658,16 @@ func (dc *DeploymentController) updateDeployment(deployment *extensions.Deployme
 	// TODO: Using client for now, update to use store when it is ready.
 	return dc.expClient.Deployments(deployment.ObjectMeta.Namespace).Update(deployment)
 }
+
+// byCreationTimestamp sorts a list by creation timestamp, using their names as a tie breaker.
+type byCreationTimestamp []*api.ReplicationController
```

Comment:
We have been reimplementing this in a bunch of places. Eventually I would love to have it as an utility and re-use it. Can you move [this piece of code](https://github.com/kubernetes/kubernetes/blob/9e66a8f1343f612a25494377df2a8b47ba6d88db/pkg/controller/replication/replication_controller_utils.go#L59-L70) somewhere under pkg/util or pkg/api and pull it from there? @bgrant0607 @bprashanth @nikhiljindal any suggestions about the final destination of that code?